### ISSUE TITLE
Logged-In User is directed to Post description page after clicking on the description of the page.

### ISSUE THREAD

**Issue by rabbi08** [FightPandemics/FightPandemics#2156]
1. Login into STG:"https://staging.fightpandemics.work/"
2. Navigate to HelpBoard without logging in.
3. Click on any 'Post description'

You will observe that the user is NOT able to click on the 'post description'

4. Now login in using valid email/pwd
5. Navigate to HelpBoard without logging in.
6. Click on any 'Post description'

Issue: You will observe that user is re-directed to 'Post Description' page.
Expected Result: User should be directed to post description page when:
1. click on 'Like' button
2. Click on 'Comment' button
3. click on 'Show more..'

*Other than that -> user should be directed to 'User profile' after clicking on user profile name and logo.

Please see below GIF.
![ezgif com-video-to-gif-9](https://user-images.githubusercontent.com/67168400/104860861-34c08580-58fb-11eb-9376-228745d59e8c.gif)

**Comment by rabbi08** [comment]
@galeza great if you can confirm/comment on the expected behavior in above issue scenario, i.e. user should not be directed to post description page after clicking on description of the post. 

Thanks

**Comment by galeza** [comment]
hi @rabbi08 
Number 5 is "Navigate to HelpBoard without logging in." - maybe we should have **with** login? My understanding is that steps 1-3 are for a user that is not logged in, steps 4-6 for logged in user.
I compared this behaviour with Twitter. When I'm NOT logged in I can click on a tweet and I'm redirected to the Tweet page. But when I click Like, comment button -  I'm redirected to the Login page
My understanding is that you want to unify FP behaviour for logged in and not logged in user - do not open post page when clicking on title or post text. Right?

@CamilaArias could you look at this issue and add your thoughts? In which direction we should go?

**Comment by emanuelquerty** [comment]
I want to take this ticket but still waiting for the @galeza and @CamilaArias to agree in which direction to go

**Comment by adorapham** [comment]
@rabbi08, i agree with @galeza . I think when you click on post description , you are directed to post page, not profile of individual or organization page.  the profile of individual or organization page will display when we click on name of individual or name of organization for each post. 
on the feed -help board page, without signed in, the non-login user only is able to view post information if the post's content is short and in view. If the post content is too long -see view more , when clicking on it , it directs to login page.  it is different from twister, it navigates to twister post page and allows non-login user to view the whole post content .  FP is a bit different , non-login user seems only allowing to view post with the short content, for long content, they are expected to log-in :).  
In summary , clicking on post description will navigate to post page, if you are non-login user , you are directed to login page .
